cmd/bacalhau/wasm_run.go

```diff
@@ -49,6 +53,15 @@ type WasmRunOptions struct {
 	NodeSelector    string // Selector (label query) to filter nodes on which this job can be executed
 	Publisher       opts.PublisherOpt
 	Inputs          opts.StorageOpt
+
+	// Engine Params
+
+	Entrypoint           string
+	ImportModules        []model.StorageSpec
+	EnvironmentVariables map[string]string
+	// EntryModule are passed as an argument over the CLI.
+	// Parameters are passed as an argument over CLI.
```

Comment:
These contain user input from the CLI. We write to these instead of the `EngineSpec` directly.